theseus_gui/src/components/ui/InstanceCreationModal.vue

```diff
@@ -0,0 +1,205 @@
+<template>
+  <Modal ref="modal" header="Create Instance">
+    <div v-if="showContent" class="modal-body">
+      <div class="image-upload">
+        <Avatar :src="display_icon" size="md" :rounded="true" />
+        <div class="image-input">
+          <Button @click="upload_icon()">
+            <UploadIcon />
+            Upload Icon
+          </Button>
+          <Button @click="reset_icon">
+            <XIcon />
+            Remove Icon
+          </Button>
+        </div>
+      </div>
+      <div class="input-row">
+        <p class="input-label">Game Version</p>
+        <DropdownSelect v-model="game_version" :options="game_versions" />
+      </div>
+      <div class="input-row">
+        <p class="input-label">Loader</p>
+        <Chips v-model="loader" :items="loaders" />
+      </div>
+      <div class="input-row">
+        <p class="input-label">Loader Version</p>
+        <Chips v-model="loader_version" :items="['latest', 'stable', 'other']" />
+      </div>
+      <div v-if="loader_version === 'other'">
+        <div v-if="game_version" class="input-row">
+          <p class="input-label">Select Version</p>
+          <DropdownSelect v-model="specified_loader_version" :options="selectable_versions" />
+        </div>
+        <div v-else class="input-row">
+          <p class="warning">Select a game version before you select a loader version</p>
+        </div>
```

Comment:
This looks good, but I do wonder if it would be better to hide this behind an "Advanced" toggle. Most users probs don't care about the loader version/don't understand how it works. 

We could also have an "include snapshots" selector there too for people to be able to create snapshot GVs